warehouse/manage/views.py

```diff
@@ -733,6 +753,20 @@ def delete_project_role(project, request):
                     submitted_from=request.remote_addr,
                 ),
             )
+            owners = (
+                request.db.query(Role)
+                .join(Role.user)
+                .filter(
+                    Role.role_name == 'Owner',
+                    Role.project == role.project,
+                )
+            )
+            owner_emails = [owner.user.email for owner in owners
+                            if owner.user.email]
+
+            send_removed_from_role_email(request, role)
+            if owner_emails:
```

Comment:
Can we move this condition into the `send_role_removed_from_user_email` function instead?